src/config/serverConfigDefaults.ts

```diff
@@ -197,6 +197,8 @@ const ServerConfigDefaults: Partial<IServerConfig> = {
         'The study is unauthorized. You need to request access.',
     comparison_categorical_na_values: 'NA',
     skin_hide_download_controls: false,
+
+    oncoprint_clinical_tracks_show_by_default: '',
```

Comment:
@pvannierop why not boolean?

Reply:
We decided to rename to _oncoprint_clinical_tracks_tracks_config_json_.

Reply:
`oncoprint_clinical_tracks_tracks_config_json` should be `oncoprint_clinical_tracks_config_json`, @pvannierop ?

Reply:
Backend changes: https://github.com/cBioPortal/cbioportal/pull/9709

Reply:
solved in [73a1b59](https://github.com/cBioPortal/cbioportal-frontend/pull/4152/commits/73a1b59bcea5cf98e8d83b39447d1eb8e5470638)